### PR TITLE
Tickets/dm 34702

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -32,9 +35,10 @@ test:
     - tests
     - schema
     - setup.cfg
+    - pyproject.toml
     - .git
-  commands:
-    - pytest
+#  commands:
+#    - pytest
```
Comment (r-owen) [inline conda/meta.yaml:41]: Please remove these comment-out lines, if practical. If you want to keep them then please add a comment explaining why they are commented-out and when they can be uncommented (or discarded), preferably with a JIRA ticket #.

File: python/lsst/ts/salobj/cli.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# This file is part of ts_salobj.
```
Comment (r-owen) [inline python/lsst/ts/salobj/cli.py:1]: Please delete this file. It is no longer being used.

**Review by r-owen — APPROVED** [review]
Looks great. I had a few small suggestions.

File: conda/meta.yaml (hunk 1 above)
Comment (wvreeven) [reply conda/meta.yaml:41]: Thanks for catching that. Those should not be uncommented. I just did it to be able to run the build quicker. I will uncomment them again.

File: python/lsst/ts/salobj/cli.py (hunk 2 above)
Comment (wvreeven) [reply python/lsst/ts/salobj/cli.py:1]: Done

**Pull request merged**